<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>patientAppoint</class>
 <widget class="QWidget" name="patientAppoint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>568</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>879</width>
    <height>568</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>879</width>
    <height>568</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>预约挂号</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/* 主窗口样式 */
QWidget {
    font-family: &quot;Segoe UI&quot;, &quot;Microsoft YaHei&quot;, sans-serif;
    background-color: #f8fafc;
}

/* 标签样式 */
QLabel {
    background-color: transparent;
    color: #374151;
    font-size: 14px;
    font-weight: 500;
    padding: 4px 0px;
}

/* 数字选择框样式 */
QSpinBox {
    background-color: white;
    border: 1px solid #e2e8f0;
    border-radius: 8px;
    padding: 8px 12px;
    font-size: 14px;
    color: #1f2937;
    min-height: 30px;
    max-width: 100px;
}

QSpinBox:focus {
    border: 1px solid #4a90e2;
    background-color: #f0f7ff;
    outline: none;
}

QSpinBox:hover {
    border: 1px solid #cbd5e1;
}

QSpinBox::up-button, QSpinBox::down-button {
    width: 20px;
    border-left: 1px solid #e2e8f0;
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
    background-color: #f1f5f9;
}

QSpinBox::up-button:hover, QSpinBox::down-button:hover {
    background-color: #e2e8f0;
}

QSpinBox::up-arrow, QSpinBox::down-arrow {
    width: 8px;
    height: 8px;
}

/* 下拉框样式 */
QComboBox {
    background-color: white;
    border: 1px solid #e2e8f0;
    border-radius: 8px;
    padding: 8px 12px;
    font-size: 14px;
    color: #1f2937;
    min-height: 30px;
    min-width: 100px;
}

QComboBox:focus {
    border: 1px solid #4a90e2;
    background-color: #f0f7ff;
}

QComboBox:hover {
    border: 1px solid #cbd5e1;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left: 1px solid #e2e8f0;
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
}

QComboBox::down-arrow {
    image: none;
    border-left: 4px solid transparent;
    border-right: 4px solid transparent;
    border-top: 6px solid #64748b;
}

/* 按钮样式 */
QPushButton {
    background-color: #4a90e2;
    color: white;
    border: none;
    border-radius: 8px;
    padding: 10px 20px;
    font-size: 14px;
    font-weight: 500;
    transition: all 0.2s ease;
    min-width: 100px;
    min-height: 40px;
}

QPushButton:hover {
    background-color: #3a7bc8;
    transform: translateY(-1px);
}

QPushButton:pressed {
    background-color: #2c6bb6;
    transform: translateY(1px);
}

QPushButton:disabled {
    background-color: #cbd5e1;
    color: #94a3b8;
}

/* 查询按钮特殊样式 */
#pushButton {
    background-color: white;
    color: #4a90e2;
    border: 1px solid #4a90e2;
}

#pushButton:hover {
    background-color: #4a90e2;
    color: white;
}

#pushButton:pressed {
    background-color: #3a7bc8;
    border-color: #3a7bc8;
}

/* 预约按钮特殊样式 */
#commitDoctorPushButton {
    background-color: #10b981;
    color: white;
}

#commitDoctorPushButton:hover {
    background-color: #0d966c;
}

#commitDoctorPushButton:pressed {
    background-color: #0b845a;
}

/* 列表控件样式 */
QListWidget {
    background-color: white;
    border: 1px solid #e2e8f0;
    border-radius: 8px;
    padding: 4px;
    font-size: 13px;
    outline: none;
}

QListWidget::item {
    background-color: white;
    border-bottom: 1px solid #f1f5f9;
    padding: 12px;
    border-radius: 6px;
}

QListWidget::item:selected {
    background-color: #e3eef9;
    color: #2c6bb6;
    border-left: 3px solid #4a90e2;
}

QListWidget::item:hover {
    background-color: #f8fafc;
}

/* 滚动条样式 */
QScrollBar:vertical {
    border: none;
    background-color: #f1f5f9;
    width: 10px;
    margin: 0;
    border-radius: 5px;
}

QScrollBar::handle:vertical {
    background-color: #cbd5e1;
    border-radius: 5px;
    min-height: 30px;
}

QScrollBar::handle:vertical:hover {
    background-color: #94a3b8;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    height: 0;
}

/* 布局容器样式 */
#horizontalLayoutWidget, #horizontalLayoutWidget_2,
#horizontalLayoutWidget_3, #horizontalLayoutWidget_4 {
    background-color: white;
    border-radius: 12px;
    border: 1px solid #e2e8f0;
    box-shadow: 0 4px 6px rgba(0, 0, 0, 0.05);
}

/* 图片样式 */
QLabel[scaledContents=&quot;true&quot;] {
    border-radius: 8px;
}

#label_5 {
    opacity: 0.1;
}
   </string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>381</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>请选择日期：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="yearSpinBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>48</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>126</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="minimum">
       <number>2025</number>
      </property>
      <property name="maximum">
       <number>2027</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="monthSpinBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>48</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>126</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>12</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="daySpinBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>48</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>126</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>31</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>381</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>请选择时间：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>48</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>126</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="maximum">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_2">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>48</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>126</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="maximum">
       <number>60</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>211</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>请选择科室</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <property name="minimumSize">
       <size>
        <width>126</width>
        <height>48</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>142</width>
     <height>62</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>142</width>
     <height>62</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>302</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QComboBox" name="comboBox_2">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>48</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>-50</y>
     <width>71</width>
     <height>721</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources.qrc">:/resources/icons/1注册登录d.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>161</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources.qrc">:/resources/icons/3医生个人信息.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="availableDoctorListWidget">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>150</y>
     <width>300</width>
     <height>360</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>360</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>360</height>
    </size>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="movement">
    <enum>QListView::Movement::Static</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="commitDoctorPushButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>140</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>预约</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
